docs/guides/layers.md

```diff
@@ -22,10 +22,10 @@ viewer.add_image(data, name='astronaut')
 ```
 
 To learn more about the layers available, see the
-[Layers documentation](napari.layers). To learn about how to use the layers currently supported by napari, check out the
-[Using layers how-to guides](../../howtos/layers/index). For a gentle
+{class}`~napari.layers`. To learn about how to use the layers currently supported by napari, check out the
```

Comment:
Can we make this a regular link like in the original? it looks wierd:
<img width="499" alt="image" src="https://github.com/napari/docs/assets/76622105/094fad87-b741-4754-8e2e-51ea34e2dd26">

Reply:
Done